### ISSUE TITLE
Helper method for list of dummies

### ISSUE THREAD

**Issue by jamiehumphries** [FakeItEasy/FakeItEasy#675]
I really like the `Dummy` feature of FakeItEasy. One thing that I find myself needing to do on every project is to create a collection of dummies (e.g. for seeding a test database).

I always end up writing an extension method that give me a fluent syntax for this, like:
```cs
IEnumerable<User> users = 42.Dummy<User>();
```
the code for which is just something like:
```cs
public static class DummyHelpers
{
    public static IEnumerable<T> Dummy<T>(this int count)
    {
        for (var i = 0; i < count; i++)
        {
            yield return A.Dummy<T>();
        }
    }
}
```

Would you consider adding something like this to the framework? If so I would be happy to write up a pull request with any necessary tests.

---

- [x] implement - ~~#702~~
- [x] approve API change - ~~#705~~
- [ ] update docs - #710
- [x] update https://github.com/FakeItEasy/FakeItEasy/wiki/Changes-in-version-2.0

**Comment by blairconrad** [comment]
Hi, @jamiehumphries. Nice to see you again.
Looks like you're describing `A.CollectionOfFake`, but for dummies, yes?
If something were to be added, I'd expect it should look like that, rather than your syntax...

**Comment by jamiehumphries** [comment]
Ah, yes. I hadn't spotted `A.CollectionOfFake`, but yes that is the idea :smile:

The syntax isn't particularly important to me, I just find the fluent `int` extension form quite readable.

Would  `A.CollectionOfDummy` be considered as a new feature?

**Comment by blairconrad** [comment]
@jamiehumphries, I don't see why not. As noted, it parallels `CollectionOfFake`, and is unlikely to be confused with anything else.

**Comment by adamralph** [comment]
I'm a bit hesitant to add extension methods on primitive types but I'm perfectly happy with `A.CollectionOfDummy`. :+1:

**Comment by adamralph** [comment]
Taken by @t0mburton

**Comment by adamralph** [comment]
@jamiehumphries [thanks](https://github.com/FakeItEasy/FakeItEasy/pull/702) to @t0mburton, this will be in the 2.0 release!

**Comment by adamralph** [comment]
Needs #705 to be merged before we can close.

**Comment by jamiehumphries** [comment]
Excellent - thanks @adamralph and @t0mburton!

**Comment by blairconrad** [comment]
I may have missed it, @adamralph, but do we have readthedocs docs and changed in 2.0 notes?

**Comment by adamralph** [comment]
@blairconrad good catch. They are not added.

**Comment by adamralph** [comment]
I've added a check list to the issue description.

**Comment by thomaslevesque** [comment]
> Needs #705 to be merged before we can close.

Done

**Comment by adamralph** [comment]
Still needs a docs update.

**Comment by blairconrad** [comment]
@t0mburton, thanks very much for your work on this issue. Look for your name in the release notes. :trophy:

This issue has been fixed in [release 2.0.0](https://github.com/FakeItEasy/FakeItEasy/releases/tag/2.0.0).
